backend/src/routes/v0/auth.py

```
@@ -1,46 +1,21 @@
 from typing import Annotated, Literal
-from fastapi import Body, HTTPException, Request, status, Depends, APIRouter
-from fastapi.responses import JSONResponse, UJSONResponse
-from sqlalchemy import text
-from sqlalchemy.orm import Session
+from fastapi import Body, HTTPException, status, Depends, APIRouter
+from fastapi.responses import UJSONResponse
 from sqlalchemy.ext.asyncio import AsyncSession
 from sqlalchemy.future import select
```

Ruff (0.11.9)

`sqlalchemy.future.select` imported but unused

Remove unused import: `sqlalchemy.future.select`

(F401)